cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -91,17 +100,20 @@ def test_vm_console(request, appliance, setup_provider, context, configure_webso
                 # Enter Password:
                 vm_console.send_keys("{}".format(console_vm_password))
                 logger.info("Wait to get the '$' prompt")
-                vm_console.wait_for_text(text_to_find=catalog_item.provider.data.templates.
-                    console_template.prompt_text, timeout=200)
+                if not provider.one_of(OpenStackProvider):
+                    vm_console.wait_for_text(text_to_find=catalog_item.provider.data.templates.
+                        console_template.prompt_text, timeout=200)
+                else:
+                    time.sleep(15)
 
                 def _validate_login():
                     # the following try/except is required to handle the exception thrown by SSH
                     # while connecting to VMware VM.It throws "[Error 104]Connection reset by Peer".
                     try:
-                        user_count_after_login = vm_ssh_client.run_command("who --count",
+                        user_count_after_login = vm_ssh_client.run_command(ssh_who_command,
```

Comment:
Please indent as described in our [Style Guide](http://cfme-tests.readthedocs.io/guides/dev_guide.html#code-style).